packages/sanity/src/core/form/inputs/DateInputs/base/calendar/Calendar.tsx

```diff
@@ -342,25 +340,10 @@ function CalendarMonthSelect(props: {
     goToNextMonth: string
   }
 }) {
-  const {moveFocusedDate, onChange, value, labels, monthNames} = props
-
-  const handlePrevMonthClick = useCallback(() => moveFocusedDate(-1), [moveFocusedDate])
-
-  const handleNextMonthClick = useCallback(() => moveFocusedDate(1), [moveFocusedDate])
+  const {onChange, value, labels, monthNames} = props
```

Comment:
Shall we remove the labels from our props too? They were only used for the Button component, as far as I can see.
```suggestion
  const {onChange, value, monthNames} = props
```

Github won't let me remove the labels parameter from the function arguments here or in codespaces unfortunately.